### DISCUSSION TITLE
Optionless Block Previews

### DISCUSSION THREAD

**Discussion by freshleafmedia** [area17/twill#1639] [Q&A]
We have a block which has no fields, it's just a piece of static content users can drop into an editor, which works fine however the admin doesn't show a preview for it.

I believe the preview assumes that if a block has no values that the block is new and only once the user starts populating values does it attempt to render.

Is this correct and can this be changed to take into account if there are no fields at all rather than no unpopulated fields?

**Comment by ifox** [comment]
Hi @freshleafmedia, I believe a hidden input field with a value should work for this use case, but we can definitely look into improving that.
